lib/java/com/google/android/material/divider/res/values/attrs.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2020 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+<resources>
+
+  <!-- Style to use for Dividers in this theme, usually to be used as a view. -->
+  <attr name="dividerStyle" format="reference" />
+
+  <declare-styleable name="Divider" />
```

Comment:
Could this styleable include `<attr name="thickness" format="dimension" />` which could then be parsed/applied in the `MaterialDivider` class instead of using the hardcoded dimen? I also think the general name should change from "thickness" to "height"

Reply:
@ricknout I originally allowed for this behavior, but then realized that developers are required to define the `android:layout_height` attribute. It would be confusing to have two different height attributes, it is simpler to just have a multi-purpose `android:layout_height`:

* If the developer wants to use Material's default divider height, they set the divider height to `WRAP_CONTENT`
* If the developer wants to customoize the height, they can set `android:layout_height` to a custom height.

Reply:
Fair enough. Although I still find "thickness" to be a word that is not well known on Android in general. Perhaps "size" would be better?

Reply:
Yep, that seems better